content/news/2023-12-23-galaxy-sc-2023-in-review/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Galaxy Single-cell Community: Year in Review"
+date: '2023-12-23'
+tease: 'The Galaxy Single-cell Community provides a 2023 review!'
+authors: Wendi Bacon, Pavankumar Videm, Mehmet Tekman
+authors_structured:
+- github: nomadscientist
+- github: pavanvidem
+- github: mtekman
+tags: [gtn, communications, single-cell]
+subsites: [all-eu,global,esg]
+---
+
+<div class="float-right" style="max-width: 400px">
+
+![swirled cluster dots surround a circle of people all holding hands, looking towards the bright center (future)](./2023_dec_sc.png)
+
+</div>
+
+🚀Embarking on a cosmic journey, the Galaxy Single-cell Community has clustered together to unveil a constellation of tools, making strides in
+RNA-stellar discoveries and creating out-of-this-world workflows. With a commitment to battling work duplication across the multiverse,
+this community is boldly charting a course for global domination, proving that when it comes to bioinformatics, the Galaxy is the limit!✨
+
+# Tools & Workflows
+
+The past year has been a spectacular journey in advancing tools and workflows within the Galaxy Single-cell Community.
+Breaking free from the confines of scRNA-seq, we've expanded our horizons to include tools for analyzing data from other modalities such
+as single-cell ATAC-seq and CITE-Seq. Dive into our updated scRNA-seq analysis tools, now featuring more downstream analysis options for
+trajectory analysis. We've even thrown in advanced analysis tools for estimating cell type proportions from bulk RNA-seq data!
+
+**RNA STARSolo** just got a makeover to version 2.7.10b, boasting a revamped UI, Velocyto-like UMI counting, and nifty alternatives for
+quantifying single nuclei data. Analyzing single-cell ATAC-seq data on Galaxy is now a breeze with the introduction of
+the **EpiScanpy** tool suite, built on the renowned **Scanpy** toolkit. Plus, we've added some powerful tools from the **Sinto** toolkit
+for preprocessing scATAC-seq data on Galaxy. And that's not all! The **MuSiC** tool suite in Galaxy now unveils deconvolution tools
+for discovering cell type composition in bulk RNA-seq using cell types within scRNA-seq. Multi-faceted comparisons are now at your fingertips with the **MuSiC Compare** tool.
+
+Seurat fans, rejoice! Updates have rolled in, offering additional functionalities and optional CITE-seq capabilities at runtime.
+
+# Community Infrastructure
+
+Our efforts in community building have seen the unification of single-cell subdomains into [https://singlecell.usegalaxy.eu/](https://singlecell.usegalaxy.eu/),
+the creation of the Single Cell Community of Practice hub page ([https://galaxyproject.org/projects/singlecell/](https://galaxyproject.org/projects/singlecell/)),
+and a user-focused Matrix channel. Testing brilliant widgets, automated updates, and the latest addition of automated help request updates onto
+Matrix channels have made community collaboration a breeze.
```

Comment:
Link to @mtekman blog post about it?

Maybe also a link or a button to the Matrix channel?